examples/ssao/main.js

```diff
@@ -0,0 +1,701 @@
+( function () {
+    'use strict';
+
+    var ExampleOSGJS = window.ExampleOSGJS;
+
+    var OSG = window.OSG;
+    var osg = OSG.osg;
+    var osgViewer = OSG.osgViewer;
+    var osgShader = OSG.osgShader;
+    var osgUtil = OSG.osgUtil;
+    var Texture = osg.Texture;
+
+    var $ = window.$;
+    var P = window.P;
+
+    // Generates MAX_MIP_LEVEL depth mipmaps.
+    // Do not forget to change the code in ssaoFragment.glsl
+    // because the mipmaps are passed manually using texture units
+    var MAX_MIP_LEVEL = 5;
+
+    var shaderProcessor = new osgShader.ShaderProcessor();
+
+    // Loads a glTF model and update
+    // the current displayed scene in the Example
+    var loadFiles = function ( example, files, sceneName ) {
+
+        var promise = OSG.osgDB.Registry.instance().getReaderWriterForExtension( 'gltf' )
```

Comment:
I see. Is there a good reason for breaking the API? I mean, that's the way OSG works so, why don't you call the readNodeURL sequentially for each file?